### PR TITLE
Add Slovakia holidays

### PR DESCRIPTION
**Pull request by patressz**
I did have a problem with encoding, so i add test snapshot with special characters.

### CODE REVIEW

File: tests/.pest/snapshots/Countries/SlovakiaTest/it_can_calculate_slovak_holidays.snap
Comment (matusk) [inline tests/.pest/snapshots/Countries/SlovakiaTest/it_can_calculate_slovak_holidays.snap:31]:
```suggestion
        "name": "V\u00fdro\u010die Slovensk\u00e9ho n\u00e1rodn\u00e9ho povstania",
```

File: src/Countries/Slovakia.php
Comment (matusk) [inline src/Countries/Slovakia.php:25]:
```suggestion
```
this is not public holiday

File: tests/.pest/snapshots/Countries/SlovakiaTest/it_can_calculate_slovak_holidays.snap
Comment (matusk) [inline tests/.pest/snapshots/Countries/SlovakiaTest/it_can_calculate_slovak_holidays.snap:45]:
```suggestion
```
this is not public holiday

**Review by matusk — CHANGES_REQUESTED** [review]

File: src/Countries/Slovakia.php
Comment (Nielsvanpach) [inline src/Countries/Slovakia.php:36]: Could you use `$this->easter($year)` instead?

**Review by Nielsvanpach — COMMENTED** [review]
Thanks! PR looks good. I'll wait on some approvals by others before merging.

File: src/Countries/Slovakia.php
Comment (patressz) [reply src/Countries/Slovakia.php]: Of course, fixed.

**Review by RobiNN1 — APPROVED** [review]

**Comment by Nielsvanpach** [comment]
Thanks!

**Pull request merged**
